language/diem-framework/modules/AccountCreationScripts.move

```diff
@@ -20,14 +20,16 @@ module AccountCreationScripts {
     /// Parent VASP account. The child account will be recorded against the limit of
     /// child accounts of the creating Parent VASP account.
     ///
-    /// ## Events
-    /// Successful execution with a `child_initial_balance` greater than zero will emit:
-    /// * A `DiemAccount::SentPaymentEvent` with the `payer` field being the Parent VASP's address,
-    /// and payee field being `child_address`. This is emitted on the Parent VASP's
-    /// `DiemAccount::DiemAccount` `sent_events` handle.
-    /// * A `DiemAccount::ReceivedPaymentEvent` with the  `payer` field being the Parent VASP's address,
-    /// and payee field being `child_address`. This is emitted on the new Child VASPS's
-    /// `DiemAccount::DiemAccount` `received_events` handle.
+    /// # Events
+    /// Successful execution will emit:
+    /// * A `DiemAccount::CreateAccountEvent` with the `created` field being `child_address`,
+    /// and the `rold_id` field being `Roles::CHILD_VASP_ROLE_ID`. This is emitted on the
+    /// `DiemAccount::AccountOperationsCapability` `creation_events` handle.
+    /// Successful execution with a `child_initial_balance` greater than zero will additionaly emit:
```

Comment:
Nit: newline above this line so that it isn't included in the bullet point for `CreateAccountEvent`

Reply:
I fixed it. Thanks!